src/converters/cover.ts

```diff
@@ -151,11 +174,14 @@ class CoverHandler implements ServiceHandler {
       getOrAddCharacteristic(this.service, hap.Characteristic.HoldPosition).on('set', this.handleSetHoldPosition.bind(this));
     }
 
-    if (exposesCanBeGet(this.positionExpose)) {
+    if (exposesCanBeGet(this.positionExpose) && !modern) {
       this.updateTimer = new ExtendedTimer(this.requestPositionUpdate.bind(this), 4000);
     }
     this.waitingForUpdate = false;
     this.ignoreNextUpdateIfEqualToTarget = false;
+    this.setTargetPositionHandled = false;
+    this.motorState = 'none';
+    this.motorStatePrevious = 'none';
```

Comment:
I'd probably use `undefined` for the undefined / unknown state, instead of defining another constant.

```suggestion
    this.motorState = undefined;
    this.motorStatePrevious = undefined;
```